libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -596,6 +598,29 @@ bool RCOutput::mode_requires_dma(enum output_mode mode) const
 #endif //#ifdef DISABLE_DSHOT
 }
 
+void RCOutput::print_group_setup_error(pwm_group &group, const char* error_string)
+{
+#ifndef HAL_NO_UARTDRIVER
+    uint8_t min_chan = 255;
+    uint8_t max_chan = 0;
+    for (uint8_t j = 0; j < 4; j++) {
+        uint8_t chan = group.chan[j];
+        if (chan >= max_channels) {
+            continue;
+        }
+        chan += chan_offset;
+        min_chan = MIN(min_chan,chan);
+        max_chan = MAX(max_chan,chan);
+    }
+
+    if (min_chan == max_chan) {
+        GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "Chan %i, %s: %s",min_chan+1,get_output_mode_string(group.current_mode),error_string);
```

Comment:
This probably needs some modification to be meaningful to a user

Reply:
At least there is some indication that it tried and failed vs not even trying. The main one I was interested in was the buffer allocation, for LEDs that generally means if you take away some LEDs it will start working .